frontend/src/features/admin-form/create/workflow/components/WorkflowContent/WorkflowContent.tsx

```diff
@@ -22,6 +45,7 @@ export const WorkflowContent = (): JSX.Element | null => {
         ))}
         <NewStepBlock />
       </Stack>
+      <DefaultOutcomeNotificationBlock />
```

Comment:
Should this be also conditionally rendered if `form.admin.betaFlags.mrfEmailNotifications`?

Reply:
Added in 7f06a83ea